python/lsst/sims/GalSimInterface/galSimCelestialObject.py

```diff
@@ -40,6 +40,10 @@ def __init__(self, galSimType, sed, ra, dec, xPupil, yPupil,
         @param [in] positionAngle is the position angle of the object in radians
 
         @param [in] sindex is the sersic index of the object
+
+        @param [in] fluxDict is a dict of photon count (not ADU) values keyed to bandpass names,
```

Comment:
Is this correct? fluxDict was derived up above as:
  `flux_dict[bb] = adu*self.photParams.gain`
so doesn't that just mean that flux_dict is electron count of the ccd which is related to photon count by quantum efficiency value?